nextjs-ssr-react-query/apps/button-e2e/cypress.config.ts

```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'cypress';
```

Comment:
Do you really need this e2e folders? 
`button-e2e`
`header-e2e`
`host-e2e`
`pip-e2e`

Reply:
No, they are not important, I removed it here, thanks :handshake: :rocket: